### ISSUE TITLE
[Suggestion] Eschew descriptions on cards in favor of leading questions.

### ISSUE THREAD

**Issue by JayLoomis** [astralfrontier/flip-a-card#55]
I've put a lot of thought into the general format of cards, especially in light of our back-and-forth on encounter cards. Here's what I've come up with.

**Observations**
1. The text on the cards is very busy and difficult to parse at a glance (even when trimmed of overflow).
2. The descriptions often offer insights into how the card might be used, but are sometimes frustrating, because they illuminate one possibility while omitting others, which sends a slanted message to players.
3. Explanation can pull the focus away from the character or situation at hand, rather than prompt creative contribution.

**Suggestion**
I think the cards might be more effective if, instead of explanatory text, they included questions to prompt players to fill in the details themselves. Let's look at a couple of examples:

Elemental
"Earth sprites. They have the ore we seek." -- Tana

This encounter involves powerful elemental forces.

- Is it a hazard or an animate entity (e.g. volcano vs. lava monster)?
- What element does it represent, and what dangers does that imply?
- How can this aspect challenge the PCs?

In action, an Elemental aspect might:
- Create, animate, or manipulate its element in useful or dangerous ways
- Change the environment with aspects of its element, like rivers of fire or barriers of wind
- Control its element in other things, like metal in electronics or water in blood

Elemental (Back)
"They had some nice gemstones too!" -- Basler

Elemental aspects have predictable natures you can exploit.

- Is it vulnerable to another element?
- Does it require certain conditions to persist (e.g. fuel for fire)?
- Can you predict its effects based on its element?

(ed. This flipside example is one where the the prompt list as written is just a distillation of the descriptive text. It's my opinion that the list of prompts is all you need. This is possibly true of all encounter cards, though I haven't thoroughly tested that notion.)

---

Upbringing: City

You grew up in a population center.

- What did living cheek-by-jowl teach you about people?
- What did the culture and social structure of city life teach you?
- What have you seen that country folk haven't?

In action, your City upbringing might:
- Help you navigate literal or figurative crowds of people
- Give you insight into social hierarchies
- Enable you to quickly assimilate new things or information

Upbringing: City (Back)

- What amenities of the city do you take for granted?
- What makes it hard for you to relate to country folk?
- How have you hardened yourself to survive city life?

**Comment by astralfrontier** [comment]
I feel like that's basically taking some of the "well you could be X, or you could be Y" text and rephrasing it as questions, and I'm okay with that - the options I was presenting were meant to spark creativity, and this just makes that more explicit.

On the other hand, this is a pretty sweeping rewrite for a lot of cards.

It's an approach I could try on the character role cards, which are pretty bare-bones right now. Or I could make this a Version 2 project.

**Comment by JayLoomis** [comment]
Yeah, it's food for thought, mostly. I printed out some cards yesterday and am starting to think about them as artifacts during play.
